src/main/java/net/mcreator/blockly/data/BlocklyLoader.java

```diff
@@ -29,6 +31,20 @@ public static void init() {
 		INSTANCE = new BlocklyLoader();
 	}
 
+	private static final List<String> builtinCategories = new ArrayList<>() {{

```

Comment:
Maybe pass all these categories within `Arrays.asList()` or `List.of()` as a constructor argument or add them inside the `BlocklyLoader()` constructor? Lists are not as complex as maps so I don't think they require things like this to be a separate block.

Reply:
I originally added them inside the constructor, but it caused an error in External Block Loader as the constructor wasn't finished. I'll go with the List.of option.